modules/migration-upgrading-migration-tool.adoc

```diff
@@ -12,6 +12,7 @@ If you are upgrading from {mtc-short} 1.1 to 1.2, you must update the service ac
 [id='upgrading-cam-ocp-4_{context}']
 == Upgrading {mtc-short} on an {product-title} 4 cluster
```

Comment:
This is really two procedure modules. When you split the assemblies, please also split this module.